x-pack/plugins/fleet/public/components/agent_enrollment_flyout/cloud_formation_instructions.tsx

```diff
@@ -91,24 +48,23 @@ export const CloudFormationInstructions: React.FunctionComponent<Props> = ({
         }
       )}
     >
-      <EuiText>
-        <FormattedMessage
-          id="xpack.fleet.agentEnrollment.cloudFormation.instructions"
-          defaultMessage="Sign in to your AWS cloud provider account, and switch to the region that you want to scan, then click Launch CloudFormation."
-        />
-      </EuiText>
+      <CloudFormationGuide />
       <EuiSpacer size="m" />
       <EuiButton
         color="primary"
         fill
         target="_blank"
-        iconSide="right"
-        iconType="popout"
+        iconSide="left"
+        iconType="launch"
+        fullWidth
         href={cloudFormationUrl}
       >
         <FormattedMessage
           id="xpack.fleet.agentEnrollment.cloudFormation.launchButton"
-          defaultMessage="Launch CloudFormation"
+          defaultMessage="Launch {cloudFormation}"
+          values={{
+            cloudFormation: <strong>CloudFormation</strong>,
+          }}
```

Comment:
the texts are separated here, creating a gap in their underline and uneven boldness. 
i think its better to keep it similar to the one in `x-pack/plugins/fleet/public/applications/fleet/sections/agent_policy/create_package_policy_page/single_page_layout/components/post_install_cloud_formation_modal.tsx`